bot/exts/utils/reminders.py

```diff
@@ -172,32 +172,39 @@ async def send_reminder(self, reminder: dict, expected_time: datetime = None) ->
         if not is_valid:
             # No need to cancel the task too; it'll simply be done once this coroutine returns.
             return
-
         embed = discord.Embed()
-        embed.colour = discord.Colour.blurple()
-        embed.set_author(
-            icon_url=Icons.remind_blurple,
-            name="It has arrived!"
-        )
-
-        # Let's not use a codeblock to keep emojis and mentions working. Embeds are safe anyway.
-        embed.description = f"Here's your reminder: {reminder['content']}"
-
-        if reminder.get("jump_url"):  # keep backward compatibility
-            embed.description += f"\n[Jump back to when you created the reminder]({reminder['jump_url']})"
-
         if expected_time:
             embed.colour = discord.Colour.red()
             embed.set_author(
                 icon_url=Icons.remind_red,
                 name=f"Sorry it should have arrived {time_since(expected_time)} !"
```

Comment:
While you're doing this PR, we seem to have a little bug. It's a bit broken at this line (see screenshot below). We should also change the wording to `Sorry, your reminder should have arrived at <time here> !`

Fix this up and we should be good to go!

![image](https://user-images.githubusercontent.com/15021300/130347427-144c11bc-1548-4b41-8e9a-5b9af1f4710b.png)

Reply:
Yes, it seems like you can't put discord timestamps into the author field, so it seems like we have to go with the second message :(

Reply:
Unfortunate. Please implement, either way.